src/git_utils.mli

```diff
@@ -0,0 +1,47 @@
+val gitlab_repo :
```

Comment:
This is very likely not used anywhere outside this module, so should not be exported.